packer-centos-ami.json

```diff
@@ -0,0 +1,41 @@
+{
+    "variables": {
+      
+      "region": "us-east-1",
+      "ssh_username": "centos",
+      "instance_type": "t2.medium",
+      "keypair_name": "akumar",
+      "iam_instance_profile": "hailstone-s3-upload"
+
+    },
+    "builders": [{
+      "type": "amazon-ebs",
+      "profile": "globalnet",
```

Comment:
Why are we mentioning globalnet here? This should be abstract from that project.